app/controllers/merchants_controller.rb

```diff
@@ -1,4 +1,16 @@
 class MerchantsController < ApplicationController
+
+  def show
```

Comment:
Once we merge branches, we can apply the controller filter #require_login onto the merchant actions 👍